src/databricks/labs/ucx/azure/credentials.py

```diff
@@ -251,3 +253,17 @@ def run(self, prompts: Prompts, include_names: set[str] | None = None) -> list[S
         else:
             logger.info("No Azure Service Principal migrated to UC Storage credentials")
         return execution_result
+
+    def run_access_connector_migration(self, prompts: Prompts) -> list[StorageCredentialValidationResult]:
+        plan_confirmed = prompts.confirm("Please confirm to create an access connector for each storage account.")
+        if plan_confirmed is not True:
+            return []
+
+        self._resource_permissions.create_access_connectors_for_storage_accounts()
+
+        return []
```

Comment:
`run_service_principal_migration` creates storage credentials as well. We need a tactic for avoiding permission overlap when adding an access connector for each storage account